### DISCUSSION TITLE
配置 CFW 后提示超时错误

### DISCUSSION THREAD

**Discussion by operapeking** [SeaHOH/GotoX#248] [问答/咨询] [answered]
运行时一直出现错误：
`11:01:52 W 104.16.133.31 _create_ssl_connection 'x.domain.xyz' 返回 TimeoutError('timed out')，重试`

在 Power Shell 中执行 `(iwr "https://x.domain.xyz").Content` 可以返回 `OK.`

请问这是什么问题，我需要如何配置。

**Comment by SeaHOH** [comment, marked as answer]
1. 姑且先尝试增大 `timeout` 设置。

1. 使用自定义 IP，填入绑定域名的解析结果，并禁用 `explodeip`。
